mne/io/artemis123/tests/test_artemis123.py

```diff
@@ -28,14 +26,12 @@ def test_data():
 def test_utils():
     """Test artemis123 utils."""
     # make a tempfile
-    fd, path = tempfile.mkstemp()
-    try:
-        _generate_mne_locs_file(path)
-        installed_locs = _load_mne_locs()
-        generated_locs = _load_mne_locs(path)
-        assert_equal(installed_locs, generated_locs)
-    finally:
-        os.remove(path)
+    tmp_dir = _TempDir()
+    tmp_fname = op.join(tmp_dir, 'test_gen_mne_locs.csv')
+    _generate_mne_locs_file(tmp_fname)
+    installed_locs = _load_mne_locs()
+    generated_locs = _load_mne_locs(tmp_fname)
+    assert_equal(installed_locs, generated_locs)
```

Comment:
unfortunately, installed and generate locs are dicts so assert_allclose doesn't work :(. It seems like only assert_equal supports dicts. Do you have any other ideas?